Magic™Editor.user.js

```diff
@@ -3304,10 +3305,10 @@
 
         App.pipeMods.edit = function(data) {
             App.funcs.popOriginals();
-
+            var defaultBgColor = App.selections.body.css("background-color");
             // Visually confirm edit - SE makes it easy because the jQuery color animation plugin seems to be there by default
-            App.selections.body.animate({ backgroundColor: '#c8ffa7' }, 10);
-            App.selections.body.animate({ backgroundColor: '#fff' }, 1000);
+            App.selections.body.animate({ backgroundColor: retrieveCSSVariable("--green-200") }, 10);
```

Comment:
Likewise here `--green-200` in Stacks.css is `#487455`, not `#c8ffa7`. Same question as above; these are *very* different colors from one another.

Reply:
This one strikes a balance between light and dark mode because there is no equivalent that works in both. I went with a green that is different for light mode but still usable and also works in dark mode. The colour is used as a brief highlight of the editor background to signify it has been done. It is there for just a second, so I do not think the exact shade of green is too important to keep. The point is a short signal for completion.